charts/kube-master/charts/etcd/templates/deployment.yaml

```diff
@@ -100,16 +103,16 @@ spec:
                 - /bin/sh
                 - -ec
                 - ETCDCTL_API=3 etcdctl get foo
-            initialDelaySeconds: 15
-            periodSeconds: 5
-          {{- if .Values.backup.enabled }}
+            initialDelaySeconds: 300
+            periodSeconds: 30
           readinessProbe:
-            httpGet:
-              path: /healthz
-              port: 8080
+            exec:
+              command:
+                - /bin/sh
+                - -ec
+                - ETCDCTL_API=3 etcdctl get foo
             initialDelaySeconds: 15
```

Comment:
I would lower this to 5 seconds, no need to artificially stall the readiness of a starting etcd pod

Reply:
Changed this.